hibernation-setup-tool.c

```diff
@@ -824,6 +824,28 @@ static void perform_fs_specific_checks(const char *path)
     }
 }
 
+bool check_if_system_version_is_higher(const char *version) {
+    struct utsname my_uname;
+    if(uname(&my_uname) == -1)
```

Comment:
Minor nitpick: please use `clang-format` to keep the code formatting consistent across the file.